src/platform/plugins/shared/dashboard/public/dashboard_api/panels_manager.ts

```diff
@@ -247,19 +195,21 @@ export function initializePanelsManager(
             width: customPlacementSettings?.width ?? DEFAULT_PANEL_WIDTH,
           }
         );
+
+        pushReferences(prefixReferencesFromPanel(newId, serializedState?.references ?? []));
```

Comment:
Do you want to push `[]` on purpose? This will trigger `references$.next` with the same value. How about using `if (serializedState?.references) pushReferences(prefixReferencesFromPanel(newId, serializedState?.references))`